jina/helper.py

```diff
@@ -743,11 +757,41 @@ def typename(obj):
 
 
 def rsetattr(obj, attr: str, val):
```

Comment:
`rsetattr` and `rgetattr` are not used anywhere in the codebase anymore. Let's remove them.

Reply:
Done